src/v/cloud_storage/cache_service.cc

```diff
@@ -152,6 +164,16 @@ ss::future<> cache::clean_up_cache() {
     probe.set_size(curr_cache_size);
     probe.set_num_files(candidates_for_deletion.size());
 
+    // Updating the access time tracker in case if some files were removed
+    // from cache directory by the user manually.
+    access_time_tracker tmp;
+    for (const auto& it : candidates_for_deletion) {
+        tmp.add_timestamp(
+          it.path, std::chrono::system_clock::time_point::min());
+    }
+    _access_time_tracker.remove_others(tmp);
+    _current_cache_size = curr_cache_size;
```

Comment:
can we get rid of  `curr_cache_size` and just have one `_current_cache_size`?

Reply:
fixed